torch/autograd/__init__.py

```diff
@@ -88,32 +94,27 @@ def grad(outputs, inputs, grad_outputs=None, retain_graph=None, create_graph=Fal
     ``grad_outputs`` should be a sequence of length matching ``output``
     containing the pre-computed gradients w.r.t. each of the outputs. If an
     output doesn't require_grad, then the gradient can be ``None``).
-    Gradients can be given as Tensors when one doesn't need the graph of the
-    derivative, or as Variables, in which case the graph will be created.
 
     If ``only_inputs`` is ``True``, the function will only return a list of gradients
     w.r.t the specified inputs. If it's ``False``, then gradient w.r.t. all remaining
     leaves will still be computed, and will be accumulated into their ``.grad``
     attribute.
 
     Arguments:
-        outputs (sequence of Variable): outputs of the differentiated function.
-        inputs (sequence of Variable): Inputs w.r.t. which the gradient will be
+        outputs (sequence of Tensor): outputs of the differentiated function.
+        inputs (sequence of Tensor): Inputs w.r.t. which the gradient will be
             returned (and not accumulated into ``.grad``).
         grad_outputs (sequence of Tensor): Gradients w.r.t. each output.
-            Any tensors will be automatically converted to Variables that are
-            volatile unless ``create_graph`` is ``True``. None values can be
-            specified for scalar Variables or ones that don't require grad.
-            If a None value would be acceptable for all grad_variables, then
-            this argument is optional.
+            None values can be specified for scalar Tensors or ones that don't require
+            grad. If a None value would be acceptable for all grad_tensors, then this
+            argument is optional.
         retain_graph (bool, optional): If ``False``, the graph used to compute the grad
```

Comment:
Can you also add default value here as well?

Reply:
The retain_graph default is specified.